packages/desmos-std/src/types.rs

```diff
@@ -1,13 +1,30 @@
-use crate::query_types::DesmosQueryWrapper;
+use crate::query_router::DesmosQueryRouter;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-pub type Deps<'a> = cosmwasm_std::Deps<'a, DesmosQueryWrapper>;
-pub type DepsMut<'a> = cosmwasm_std::DepsMut<'a, DesmosQueryWrapper>;
+pub type Deps<'a> = cosmwasm_std::Deps<'a, DesmosQueryRouter>;
+pub type DepsMut<'a> = cosmwasm_std::DepsMut<'a, DesmosQueryRouter>;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum DesmosRoute {
     Subspaces,
     Profiles,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct PageRequest {
+    key: Vec<u8>,
```

Comment:
Should we send back key as `String` to desmos app, as same as `PageResponse next_key` is `String`.

Reply:
@braze Finally I found a better solution which is using cosmwasm_std::Uint64 and cosmwasm_std::Binary.
The pagination stuff could be:
```rust
pub struct PageRequest {
    key: Option<Binary>,
    offset: Uint64,
    limit: Uint64,
    count_total: bool,
    reverse: bool,
}

pub struct PageResponse {
    next_key: Option<Binary>,
    total: Uint64,
}
```

Reply:
It worked fine 👍 Great Job!